scripts/scil_compute_mean_fixel_lobe_metric_from_bundles.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Given a bundle and Bingham coefficients, compute the average lobe-specific
+metric at each voxel intersected by the bundle.
+
+Bingham coefficients volume must come from scil_fit_bingham_to_fodf.py and
+lobe-specific metrics comes from scil_compute_lobe_specific_fodf_metrics.py.
```

Comment:
Can you add a sentence saying this is the script that behaves as scil_compute_mean_fixel_afd_from_bundles.py for fODFs but here for Bingham distributions, which adds the unique possibility to capture fixel-based fiber spread (FS) and fiber fraction (FF). FD from the bingham should be "equivalent" to the AFD_fixel we are used to.

Reply:
FD is the integral of the Bingham distribution, which I think is different from the AFD, being simply the evaluated SF. No?

Reply:
Yes, in theory, if you look at the paper, the AFD_fixel should also be an integral around the maxima and full-width half-max of the fODF... You can leave as is.